basics/hello-solana/native/tests/test.ts

```diff
@@ -1,45 +1,32 @@
 import {
-    Connection,
-    Keypair,
-    sendAndConfirmTransaction,
-    Transaction,
-    TransactionInstruction,
+  PublicKey,
+  Transaction,
+  TransactionInstruction,
 } from '@solana/web3.js';
+import { start } from 'solana-bankrun';
+import { describe, test } from 'node:test';
 
-function createKeypairFromFile(path: string): Keypair {
-    return Keypair.fromSecretKey(
-        Buffer.from(JSON.parse(require('fs').readFileSync(path, "utf-8")))
-    )
-};
+describe('hello-solana', async () => {
+  // load program in solana-bankrun
```

Comment:
I can see you're running prettier over these which is awesome, but I have a small suggestion:

 - Make 1 PR with `Run prettier over all TS files. No changes`
 - Make a second PR with this work. 
 
 This will make the real changes in the PR much easier to see since there won't be formatting changes everywhere.